libraries/helpers.rb

```diff
@@ -0,0 +1,18 @@
+module HabitatCookbook
+  module Helpers
+    def hab(*command)
+      begin
+        if defined?(shell_out_with_timeout!)
```

Comment:
also you should probably use `clean_array` in favor of `a_to_s`

https://github.com/chef/chef/blob/master/lib/chef/mixin/shell_out.rb#L91-L109
https://github.com/chef/chef/blob/master/lib/chef/provider/user/linux.rb#L26-L44

instead of passing a single string to shell_out that winds up passing an array which automatically fixes files-with-spaces-needing-surrounding-quotes errors (typically windows) without having to think about it or use quotes.